newsletter/_posts/2018-04-10-WLCG-HSF-Naples.md

```diff
@@ -0,0 +1,155 @@
+struck a real chord with the community.
+
+The workshop opened with Liz Sexton-Kennedy [laying out the big picture of
+the HEP science goals](https://indico.cern.ch/event/658060/contributions/2844782/attachments/1622746/2582912/ScienceGoalsWLCG-HSFworkshop2018.pdf){:target="_naples_scigoals"},
+covering the motivations that drive research in
+particle physics, astro-particle and cosmology. New facilities usher in
+the exascale era at HL-LHC, DUNE, SKA, LIGO and LSST and we need to undertake
+major R&D in software and computing to ensure this programme
+is a success.
+
+<div style="text-align:center; padding:25px"><img src ="/images/naples-ws-liz.jpg" alt="Liz opens the workshop's science programme" /></div>
+
+Eduardo Rodrigues then reviewed the [HSF Community White Paper Roadmap](https://indico.cern.ch/event/658060/contributions/2876976/attachments/1622751/2582923/2018-03-26_WLCG-HSFWorkshopNaples.pdf){:target="_naples_cwp"}
+that was [published last year](https://arxiv.org/abs/1712.06982){:target="_cwp_roadmap_arxiv"}
+and was the major driver for the workshop. He outlined each of the CWP's
+chapters along with its major R&D goals and stressed again the approach
+of working together, re-characterising CWP as *Community Wide Projects*. 
+
+After this great introduction to the workshop, and a review of the important
+technology trends and use cases, we continued with a mixture of
+plenary and parallel sessions that covered the major areas the community
+needs to invest in.
+
+Recognising the large cost of data storage and data management for WLCG,
+and anticipating the data volumes of HL-LHC, optimising our storage 
+systems is an important and challenging area for study. Consolidating storage into larger units,
+sometimes called a *data lake*, is one area to study; the anticipation
+is that much more use of the wide area network will be made in the future
+to deliver data to where it is needed. Successes from the HEP programme,
+like the [Rucio data management system](https://rucio.cern.ch/){:target="_rucio"}
+could offer a broad engagement with other sciences also facing massive
+distributed data challenges. Later technical sessions dived into the details
+and there will be a strong coupling to how we can actually characterise
+and measure the effectiveness of our workloads on our grid infrastructures,
+which the two sessions of the *Performance and Cost Modeling* group looked
+at. There will be a lot of follow up to define the real R&D goals as well
+as the way to measure success in such a complex coupled and large-scale
+system. The *Workload Management* teams look to be involved there too
+as the data processing frameworks are a key actor here. Our current systems
+are rather diverse, but there are many shared goals. Using our large 
+distributed computing systems securely, evolving how
+we authenticate and responding to threats was discussed in the *Security*
+session of the workshop.
```

Comment:
So, I did keep mention of the security session - in the end it would be bad to have only one session that was missing. It's hard to fit into the flow though, so I made it a separate paragraph.